simulator/risc_v/riscv_register/riscv_register_popular.def

```diff
@@ -0,0 +1,8 @@
+REGISTER(s0),
```

Comment:
What is the purpose of this file?

Reply:
This is file with 8 most popular registers. " rd ' ", " rs1 ' ", " rs2 ' " are able to be 3 bits long.

Reply:
Yes I see this, but I don't see that it is included anywhere...

Reply:
In the beginning of the file 'risc_v/riscv_register/riscv_register.cpp'

Reply:
I still don't see it:

https://github.com/MIPT-ILab/mipt-mips/blob/2d96b3d2e2ce04ab20e4efe540293b31722e4692/simulator/risc_v/riscv_register/riscv_register.cpp

Anyway, I do not think that file is needed for any purpose.